src/components/currency/editForm/Currency.edit.display.js

```diff
@@ -0,0 +1,18 @@
+export default [
+  {
+    type: 'textfield',
+    input: true,
+    weight: 310,
+    key: 'prefix',
```

Comment:
We already have a ```prefix``` and ```suffix``` for the text field components. We should either use different names for these or hide the ones from the display tab.

Reply:
@travist could you clarify more?

As far as I can see ‘Currency’ inherits from ‘Number’, and has differed suffix/prefix defined [here](
https://github.com/formio/formio.js/pull/866/files#diff-46b3380c5b96fc2e752f50913ef0c35cR40) ?

Reply:
Yeah OK, I think this is good.